frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/add-capacity.spec.ts

```diff
@@ -27,10 +26,6 @@ describe('OCS Operator Expansion of Storage Class Test', () => {
     cy.visit('/');
   });
 
-  afterEach(() => {
-    checkErrors();
-  });
-
```

Comment:
put a comment.

Reply:
To be added once we fix i18n issues.

Reply:
can you just comment on the code instead of removing it so we don't miss out?